test.ll

```diff
@@ -0,0 +1,30 @@
+; ModuleID = '/home/jhuang4/workspace/test.c'
```

Comment:
I guess the inclusion of this file is a mistake?

Reply:
Yeah definitely, I'll remove it